index_V1

```diff
@@ -12,14 +37,29 @@
 		
 		<style>
 			
-			.circle_1:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
-			.circle_1:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
-			.circle_2:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
-			.circle_2:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
-			.circle_3:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
-			.circle_3:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
-			.circle_4:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
-			.circle_4:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
+			html {
+				scroll-behavior: smooth;
+			}
+			
+			@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@700;900&display=swap');
+			
+			.circle_1:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_1, img {transition: 0.6s;}
+			
+			.circle_2:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_2, img {transition: 0.6s;}
+			
+			.circle_3:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_3, img {transition: 0.6s;}
+			
+			.circle_4:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_4, img {transition: 0.6s;}
+			
+			.circle_4:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_4, img {transition: 0.6s;}
+			
+			.bottom-to-top:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.bottom-to-top, img {transition: 0.6s;}
```

Comment:
Here I updated the code for the four circle divs including their animation css styling